src/status_im/contexts/settings/language_and_currency/currency/view.cljs

```diff
@@ -0,0 +1,78 @@
+(ns status-im.contexts.settings.language-and-currency.currency.view
+  (:require [quo.core :as quo]
+            [react-native.core :as rn]
+            [status-im.common.resources :as resources]
+            [status-im.contexts.settings.language-and-currency.currency.style :as style]
+            [status-im.contexts.settings.language-and-currency.currency.utils :as utils]
+            [status-im.contexts.settings.language-and-currency.data-store :as data-store]
+            [utils.debounce :as debounce]
+            [utils.i18n :as i18n]
+            [utils.navigation :as navigation]
+            [utils.re-frame :as rf]))
+
+(defn- get-item-layout
+  [_ index]
+  #js {:length 100 :offset (* 100 index) :index index})
```

Comment:
These days I fixed a bug due to the incorrect offset in a `get-item-layout` function. What if the height of `quo/category` changes or is slightly different than 100? Ideally we would use the constant 100, but also pass it to `quo/category` to merge into its container style (this is what I had to do btw in another component listing channel members).